source/draw/README.md

```diff
@@ -17,6 +17,11 @@ The destination is similar to the viewport by defining where to draw. The differ
 ## Transform
 Transform is the inverse camera matrix applied after the destination scaling.
 
+## FlatTransform
+FlatTransform is applied after the projection in 2D.
```

Comment:
I don't really like this description, it only complicates things. I would prefer something like this:
> FlatTransform is a 2D transform applied after the projection matrix, i.e. in the projected coordinate system. This can be useful when 2D-operations like translation and cropping should be applied as a final step on the rendered image.